crates/nu-std/CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+If you have questions along the way, you can post a question in your PR 
+or have a more casual discussion with Nushell fans on 
+[Discord implementation-chat channel](https://discord.com/channels/601130461678272522/615962413203718156)
+1. When you get to an appropriate state of doneness, push your changes to the PR and remove the WIP flag.
+2. Team members and other contributors will then review your PR.  
+Respond to any review comments they raise and resolve them one way or another.
+(Not all comments demand you make a change!)
+1. When you and the team are comfortable with the PR, 
+a team member will merge it into the repo and you can delete your working branch.
+2. If you've added a whole new command or made a breaking change, 
+(strongly) consider writing up a section for the release notes.  
+Currently, release notes are maintained in a different repo, [nushell.github.io](https://github.com/nushell/nushell.github.io). 
+Make your change in a local clone of that repo and submit a PR to the release notes repo to get it integrated.
+
+## Developing
+
+### Setup
+To work on the standard library, you won't necessarily need to compile or debug the interpreter itself, 
+so you might not need to install the Rust toolchain. 
+But if you want to do that, see [nushell CONTRIBUTING.md](https://github.com/nushell/nushell/blob/main/CONTRIBUTING.md).
+
+1. Clone the Nushell repo containing the standard library.  
+Currently, that's the [Nushell interpreter source repo](https://github.com/nushell/nushell).
+    ```
+    git clone https://github.com/nushell/nushell
+    ```
+
+2. Arrange a runnable Nushell interpreter  
+  Either install a [pre-built distribution](https://github.com/nushell/nushell/releases), or build from source:
+    ```
+    git clone https://github.com/nushell/nushell
+    cd nushell
+    cargo run
+    ```
+1. Focus your IDE on the standard library folder  
+(Currently, it is a Rust crate within the Nushell repo)
+
+    ```
+    cd /path/to/nushell/repo
+    code .  # opinionated: we love vscode!
+    (open folder crates/nu-std in IDE)
+   ```
+### The PR
+If your idea is mostly a bug fix, you might not open a PR till you've got a fix (nearly)ready to go.  
+
+But if you're considering new commands or substantial or (even simple) breaking changes, 
+it's good to advertise your proposed work to other collaborators for their awareness 
+and to solicit their early feedback. 
+Often, this feedback will result in a better and more maintainable solution long term, 
+so give your fellow collaborators the opportunity.  Someday, they'll return the favor...
+
+To open a PR before you've got code ready to go:
+  1. create a feature branch in your fork of the repo
+  2. make a trivial code change and push it to Github
+  4. open a PR on the standard library repo
+  5. **mark it as a work in progress (WIP)**  
+This signals that you are not yet asking to have the PR formally reviewed and merged.
```

Comment:
even better => toggle the "create draft pull request" with the arrow next to the green default "create pull request" OR click on the "still a work in progress? convert to draft" button on the status side pannel!

that would mark the PR gray as a "true" DRAFT :wink:

Reply:
"draft" PR, gotcha.  Don't want to depend on the current github UI.

Reply:
for this i'm not sure you can abstract that much away from the UI unfortunately :thinking: 
we could always use `gh` but that's probably too advanced for this document :relieved: